plugins/offboard/include/plugins/offboard/offboard.h

```diff
@@ -139,13 +186,46 @@ class Offboard : public PluginBase {
      */
     bool is_active() const;
 
+    /**
+     * @brief Set the position in NED coordinate and yaw.
+     *
+     * @see PositionNEDYaw
+     * @param position_ned_yaw Position and yaw `struct`.
+     */
+    void set_position_ned_yaw(PositionNEDYaw position_ned_yaw);
+
     /**
      * @brief Set the velocity in NED coordinates and yaw.
      *
      * @param velocity_ned_yaw Velocity and yaw `struct`.
      */
     void set_velocity_ned(VelocityNEDYaw velocity_ned_yaw);
 
+    /**
+     * @brief Set the velocity in NE and altitude in D coordinate and yaw.
+     *
+     * @see VelocityAltitudeNEDYaw
+     * @param velocity_altitude_ned_yaw Velocity/Altitude and yaw `struct`.
+     */
+    void set_velocity_altitude_ned_yaw(VelocityAltitudeNEDYaw velocity_altitude_ned_yaw);
+
+    /**
+     * @brief Set the position in NE and climbrate in D coordinate and yaw.
+     *
+     * @see PositionClimbRateNEDYaw
+     * @param position_climbrate_ned_yaw Position/Climbrate and yaw `struct`.
+     */
+    void set_position_climbrate_ned_yaw(PositionClimbRateNEDYaw position_climbrate_ned_yaw);
```

Comment:
If the type is `ClimbRate` it should be `climb_rate`. I would probably go for `Climbrate` and `climbrate`.